src/drive/web/modules/views/OnlyOffice/Toolbar/FileIcon.jsx

```diff
@@ -12,14 +12,12 @@ const FileTypeIcon = {
   text: FileTypeTextIcon
 }
 
-const FileIcon = ({ fileWithPath }) => {
-  const fileClass = fileWithPath.class
-
-  return <Icon className="u-ml-half" icon={FileTypeIcon[fileClass]} size={32} />
+const FileIcon = ({ type }) => {
```

Comment:
We should avoid to rename concept. I know what class is on an io.cozy.files , but I don't know what type is.

Reply:
`class` est un mot clé, j'ai remplacé par `fileClass`